test/utils.py

```diff
@@ -10,12 +11,14 @@
 import torch
 
 
-# For use with decoded data frames, or in other instances were we are confident that
-# reference and test tensors should be exactly equal. This is true for decoded data
-# frames from media because we expect our decoding to exactly match what a user can
-# do on the command line with ffmpeg.
+# For use with decoded data frames. On Linux, we expect exact, bit-for-bit equality. On
+# all other platforms, we allow a small tolerance.
```

Comment:
Specifically this comment doesn't explain the "why" -- only the "what".

"What" comments aren't as useful for future readers because the "what" can be redundant with the code itself.